README.rst

```diff
@@ -62,65 +176,231 @@ A simple example to get you up and running:
   print("Results:")
   print(result)
 
-Example output:
+You should see the following output printed by Python:
 
-.. code-block::
+.. code-block:: text
 
   Circuit:
   (0, 0): ───X^0.5───M('m')───
   Results:
   m=11000111111011001000
 
+Congratulations! You have run your first quantum simulation in Cirq. You can
+continue learning more by exploring the `many Cirq tutorials
+<#tutorials>`__ described below.
 
-Feature requests / Bugs / Questions
------------------------------------
 
-If you have feature requests or you found a bug, please `file them on GitHub <https://github.com/quantumlib/Cirq/issues/new/choose>`__.
+Cirq Documentation
+------------------
```

Comment:
The primary documentation source is the Cirq homepage (which seems to be missing).

Can we add it prominently as the first link in the Documentation?

Reply:
Yes, good point! I'll add it in the md format.

I just now also filled in the URL field in the GitHub project settings, so there's a link there now:

![image](https://github.com/user-attachments/assets/d200eaa0-c2cd-41d3-b133-7ed4e39d6897)